src/components/item.jsx

```diff
@@ -1,34 +1,34 @@
 import React from 'react';
 import './item.css';
 import $ from 'jquery';
+import help from './defaulthelpers.jsx';
 
-const itemDetailsUrl = 'http://localhost:3200/';
+const itemDetailsUrl = '';
+const reviewsUrl = '';
```

Comment:
These get changed from global variables to state in a future PR, correct? I think I may be doing these PRs out of order.